code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/GenDefs.hs

```diff
@@ -284,7 +284,7 @@ yForceMD_2 = mkMultiDefnForQuant quant EmptyS defns
                     mkDefiningExpr "yForceWithMass2"
                       [] EmptyS $ express $ forceGQD ^. defnExpr,
                     mkDefiningExpr "yForceWithAngle2"
-                      [] EmptyS $ express E.yForceWithAngle_2]
```

Comment:
having said that, being able to remove many many `express` calls is very nice!

Reply:
Certainly! I think that a few well-placed `ExprC`s to replace a few `Express` constraints can also be useful in further removing "earlier" `express` calls.